code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -248,6 +248,37 @@
 	new /obj/item/weapon/grenade/chem_grenade/cleaner(src)
 	new /obj/item/weapon/grenade/chem_grenade/cleaner(src)
 
+/obj/item/weapon/storage/belt/lazarus
+	name = "trainer's belt"
+	desc = "For the pokemo- mining master, holds your lazarus capsules."
+	icon_state = "lazarusbelt"
+	item_state = "lazbelt"
+	w_class = 4
+	max_w_class = 4
+	max_combined_w_class = 28

```

Comment:
have max_w_class be 1 and max_combined_w_class be 6 is enough, since each capsule has a w_class of 1 and you can only hold 6 in the belt.